src/main/java/mate/academy/internetshop/controllers/ProductAddController.java

```diff
@@ -0,0 +1,36 @@
+package mate.academy.internetshop.controllers;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.academy.internetshop.lib.Injector;
+import mate.academy.internetshop.model.Product;
+import mate.academy.internetshop.service.ProductService;
+
+@WebServlet("/addProduct")
+public class ProductAddController extends HttpServlet {
```

Comment:
```suggestion
public class AddProductController extends HttpServlet {
```

Reply:
I named it like this as it's convenient for me that it's near all product controllers so that I can search for it faster, can I leave it?

Reply:
Done